content/concepts/asset-manager.md

```diff
@@ -1238,9 +1238,14 @@ In this formula:
 The `type` and `parameters` properties are mandatory.
 :::
 
+::: info
+To do operations on your media, we always convert them to .png first, to avoid compression related losses. As a result, this conversion can increase the size of your images. To reduce your file size, we provide an `optimize jpeg` operation to convert them back to compressed jpeg files :wink:.
```

Comment:
```suggestion
To do operations on your media, we always convert them to .png first, to avoid compression related losses. As a result, this conversion can increase the size of your images. To reduce your file size, we provide an `optimize_jpeg` operation to convert them back to compressed jpeg files :wink:.
```